api/WorkletGlobalScope.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "WorkletGlobalScope": {
+      "__compat": {
+        "spec_url": "https://html.spec.whatwg.org/multipage/worklets.html#workletglobalscope",
+        "support": {
+          "chrome": {
+            "version_added": "65"
+          },
+          "chrome_android": {
+            "version_added": "65"
+          },
+          "edge": {
+            "version_added": "79"
+          },
+          "firefox": {
+            "version_added": "76"
+          },
+          "firefox_android": {
+            "version_added": "79"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "52"
+          },
+          "opera_android": {
+            "version_added": "47"
+          },
+          "safari": {
+            "version_added": "14.1"
```

Comment:
What kind of worklet is supported in Safari 14.1?

Reply:
Hmm, `AudioWorklet` it would seem: https://github.com/mdn/browser-compat-data/blob/main/api/AudioWorklet.json

I'll file an issue about Safari data for audio worklets being incomplete.